zer0-os/zOS
# Auto join conversations when loading them for the first time

### What does this do?

This joins conversations you've been invited to when you first load the conversation list.

### Why are we making this change?

Most things are done based on events but in the case where a user was invited to a conversation when the app was closed for a while you wouldn't get the invitation event as it becomes too old to be replayed. So, when loading the conversations for the first time we join any where our status is 'invite'.

### How do I test this?

With 2 accounts (A and B) close the browser for user B. From user A start a new conversation with user B. After 10 minutes open the app for user B and verify that you see the conversation in the list and can open it and send messages.
